packages/pirateship/package.json

```diff
@@ -5,39 +5,44 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "add-keys-android": "node ../../node_modules/@brandingbrand/flagship/dist/add-keys-android.js",
-    "add-keys-ios": "node ../../node_modules/@brandingbrand/flagship/dist/add-keys-ios-internal.js",
-    "compile-android": "react-native run-android --no-packager",
-    "compile-ios": "react-native run-ios --no-packager --simulator=\"iPhone X\"",
+    "add-keys-android": "node node_modules/@brandingbrand/flagship/dist/add-keys-android.js",
+    "add-keys-ios": "node node_modules/@brandingbrand/flagship/dist/add-keys-ios-internal.js",
+    "compile-android": "(test -d 'android' || yarn init android) && react-native run-android --no-packager",
+    "compile-ios": "(test -d 'ios' || yarn init ios) && react-native run-ios --simulator 'iPhone X' --no-packager",
     "compile-web": "(test -d 'web' || npm run init web) && cd ./web/ && yarn start",
     "build-ios": "./scripts/buildiOS.sh",
     "init": "flagship clean && flagship init",
     "prepare": "tsc",
+    "ios": "yarn compile-ios & yarn start",
+    "init:android": "SENTRY_WIZARD_SKIP_CONNECT=1 SENTRY_WIZARD_QUIET=1 flagship init android",
+    "init:flagship": "SENTRY_WIZARD_SKIP_CONNECT=1 SENTRY_WIZARD_QUIET=1 flagship init native",
+    "init:ios": "SENTRY_WIZARD_SKIP_CONNECT=1 SENTRY_WIZARD_QUIET=1 flagship init ios",
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "tsc:watch": "tsc -w --preserveWatchOutput",
+    "tsc:watch": "tsc -w --preserveWatchOutput --jsx react-native",
     "lint": "fscodestyle",
     "build-web-demo": "./scripts/buildWebDemo.sh"
   },
-  "dependencies": {
-    "@brandingbrand/flagship": "2.0.0",
-    "@brandingbrand/fsapp": "2.0.0",
-    "@brandingbrand/fsbazaarvoice": "2.0.0",
-    "@brandingbrand/fscategory": "2.0.0",
-    "@brandingbrand/fscommerce": "2.0.0",
-    "@brandingbrand/fscomponents": "2.0.0",
-    "@brandingbrand/fsengage": "2.0.0",
-    "@brandingbrand/fsi18n": "2.0.0",
-    "@brandingbrand/fsmockdatasources": "2.0.0",
-    "@brandingbrand/fsnetwork": "2.0.0",
-    "@brandingbrand/fsproductdetail": "2.0.0",
-    "@brandingbrand/fsproductindex": "2.0.0",
-    "@brandingbrand/fssalesforce": "2.0.0",
-    "@brandingbrand/fsshopify": "2.0.0",
-    "@brandingbrand/fsweb": "2.0.0",
+"dependencies": {
+    "@brandingbrand/flagship": "^2.0.0",
```

Comment:
Lerna handles dependencies inside of flagship so we shouldn't change the version numbers manually for references to other packages in the same repo.

Reply:
We will still need to change them if we want to copy the package outside of the repo, though. It doesn't seem like it causes any issues while inside the repo, either.